assets/scss/templates/_reportes.scss

```diff
@@ -173,6 +173,35 @@
   }
 }
 
+/*********/
```

Comment:
What is the meaning of this separator?

Reply:
If I was just generating the version to be able to do tests on the webapp.
Then I delete everything and the word neat